_posts/2018-09-27-varying-vagrant-vagrants-2-3-0.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title:  "Varying Vagrant Vagrants 2.3.0"
+date:   2018-09-27 22:00:00 -0000
+author: tomjn
+---
+
+Hi! Welcome to the release post for <a href="https://github.com/Varying-Vagrant-Vagrants/VVV/releases/tag/2.3.0">Varying Vagrant Vagrants 2.2.1</a>. For help updating, see the documentation on [keeping VVV up to date](https://varyingvagrantvagrants.org/docs/en-US/installation/keeping-up-to-date/).
```

Comment:
This should be `Varying Vagrant Vagrants 2.3.0 `

Reply:
Nice catch :p